README.md

```diff
@@ -23,10 +23,12 @@ For offline use you can download the [.zip](https://github.com/maplibre/demotile
 
 ### Contributors
 
-Kindly provided by [MapTiler](https://www.maptiler.com/) team (@klokan, @nbozon, @petr-pokorny-1, @tomasklanica).
+Kindly provided by [MapTiler](https://www.maptiler.com/) team ([@klokan](https://github.com/klokan), [@nbozon](https://github.com/nbozon), [@petr-pokorny-1](https://github.com/petr-pokorny-1), [@tomasklanica](https://github.com/tomasklanica)).  The styles & tiles for [terrain-tiles](terrain-tiles) and [tiles-omt](tiles-omt) for were added by [@acalcutt](https://github.com/acalcutt).
 
 The Open Sans Regular font PBFs were generated using the script and source fonts from https://github.com/openmaptiles/fonts.
```

Comment:
You might also get rid of the font name in this. I also added Noto Sans fonts from https://github.com/openmaptiles/fonts

So maybe just
The font PBFs were generated using the scripts and source fonts from https://github.com/openmaptiles/fonts.

Reply:
Feel free to submit a PR with the relevant changes 😀

Reply:
I added some changes I think make sense in https://github.com/maplibre/demotiles/pull/16